Doc/library/importlib.rst

```diff
@@ -420,30 +420,53 @@ ABC hierarchy::
         reloaded):
 
         - :attr:`__name__`
-            The name of the module.
+            The module's fully-qualified name.
+            It is ``'__main__'`` for non-package modules run from the file
+            system, non-package modules run from standard input and non-package
+            modules run from the module namespace.
 
         - :attr:`__file__`
-            The path to where the module data is stored (not set for built-in
-            modules).
+            The place from which the module's data was imported.  Its value and
```

Comment:
Sorry for the confusion, but in this case the technically precise "location" is better than "place".  Also, it's the module that gets imported rather than its data.  Also, the caveat about not being set is valuable here, but doesn't need to be an exhaustive list.  Finally, the rest of the text should be in the language reference.

```suggestion
            The location (usually filename) from which the module was imported.  This isn't set on all modules (e.g. built-in modules).
```

Reply:
Alright.